git_cml/git_utils.py

```diff
@@ -19,7 +19,7 @@ def get_git_repo():
     return git.Repo(os.getcwd(), search_parent_directories=True)
 
 
-def create_git_cml(repo):
+def get_git_cml(repo, create=False):
     """
     If not already created, create $git_root/.git_cml and return path
```

Comment:
Need to update docstring for this function. Can you change the description and the parameters in the comment.